rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/SQSRestServerBuilder.scala

```diff
@@ -141,7 +142,8 @@ case class TheSQSRestServerBuilder(providedActorSystem: Option[ActorSystem],
         changeMessageVisibilityBatch(p) ~
         deleteQueue(p) ~
         getQueueAttributes(p) ~
-        setQueueAttributes(p)
+        setQueueAttributes(p) ~
+        addPermissions(p)
```

Comment:
I think there's a type here: you defined the method as `addPermission` but here you call `addPermissions`